### DISCUSSION TITLE
Found Comprehensive Data source on reddit

### DISCUSSION THREAD

**Discussion by betelgeusean** [aquasarus/Starfield-Interactive-Smart-Slate#3] [General]
Adding the contents of the linked google sheet may prove useful to this project.
https://docs.google.com/spreadsheets/d/1rpOtCg_4EP86LOkCNTLFPapUrKI1afgJWKcfJXarlBI/edit#gid=912185039

**Comment by aquasarus** [comment]
Interesting! This sheet is nowhere near as searchable as Frederik's [outpost-oriented version](https://www.reddit.com/r/Starfield/comments/16g54cy/starfield_complete_list_of_resources_for_every/), but it does contain traits data. I'm gonna find a way to comb through it with a script and I'll be one step closer to supporting Traits in my app :D. Thanks!
